### ISSUE TITLE
[Bug] Set Options will disable the NOT NULL constraint

### ISSUE THREAD

**Issue by iwata** [GoogleCloudPlatform/cloud-spanner-emulator#3]
Running the following SQL against the emulator disable the `CreatedAt` NOT NULL constraint.

```sql
create table Foo(
	Id int64 not null,
	CreatedAt timestamp not null
) primary key(Id);

alter table Foo alter column CreatedAt set options(allow_commit_timestamp=true);
```

```sh
env SPANNER_EMULATOR_HOST=localhost:9010 gcloud spanner databases execute-sql test1 --instance=test-instance --sql='select table_name,column_name,is_nullable,spanner_type from information_schema.columns where table_name="Foo"'
table_name  column_name  is_nullable  spanner_type
Foo        Id           NO           INT64
Foo        CreatedAt    YES          TIMESTAMP
```

The above was tried with `gcr.io/cloud-spanner-emulator/emulator:1.0.0`.

### Expected behavior

The `CreatedAt` NOT NULL constraint remains in effect. This is the same behavior as real Spanner.

**Comment by adirastogi** [comment]
Thanks for the bug report! We've identified the fix and it will be available in the next release.

**Comment by iwata** [comment]
@adirastogi When is the next release? We are troubled by this issue.

**Comment by adirastogi** [comment]
We expect to be able to publish the next release in a couple of weeks.

**Comment by iwata** [comment]
I can confirm that the problem has been resolved in 1.1.0. Thank you.

**Comment by Vizerai** [comment]
The repository has been updated with version 1.1.0 now.
